explorer/events.go

```diff
@@ -73,6 +126,279 @@ type ChainUpdate interface {
 	ForEachV2FileContractElement(func(fce types.V2FileContractElement, created bool, rev *types.V2FileContractElement, res types.V2FileContractResolutionType))
 }
 
+// RelevantAddressesV1 returns all the relevant addresses to a V1 transaction.
+func RelevantAddressesV1(txn types.Transaction) []types.Address {
```

Comment:
`nit` we're being inconsistent with `V1/V2` suffices, sometimes it's fooV1/fooV2 and other times it's foo/fooV2 - feel free to resolve this comment though, just something to keep in the back of your head, I've noticed it a couple of times but never mentioned it, not only in `explored` but other repos as well